src/GraphQL.Tests/Bugs/Bug118SpacesInTypeNameTests.cs

```diff
@@ -51,7 +51,7 @@ public class MyInput : InputObjectGraphType
     {
         public MyInput()
         {
-            Name = "MyInput ";
+            Name = "MyInput"; // changed from "MyInput "
```

Comment:
This whitespace wasn't the root cause of #118 so should now delete it to pass the test.